src/simplewallet/simplewallet.cpp

```diff
@@ -4414,13 +4453,7 @@ boost::optional<epee::wipeable_string> simple_wallet::new_wallet(const boost::pr
   crypto::secret_key recovery_val;
   try
   {
-    recovery_val = m_wallet->generate(m_wallet_file, std::move(rc.second).password(), recovery_key, recover, two_random, create_address_file);
-    message_writer(console_color_white, true) << tr("Generated new wallet: ")
```

Comment:
Why did you comment out these lines for showing recovered wallet's address + viewkey?